edge-util/src/Microsoft.Azure.Devices.Edge.Storage/SerDeExtensions.cs

```diff
@@ -20,6 +20,20 @@ public static T FromJson<T>(this string json)
                 : JsonConvert.DeserializeObject<T>(json);
         }
 
+        public static T FromJson<T>(this string json, JsonSerializerSettings jsonSerializerSettings)
```

Comment:
This new function is mostly a duplicate of the existing `FromJson<T>` method. Could we reduce duplication by adding the `jsonSerializerSettings` argument to the original method, but with a default value? I _think_ that calling `JsonConvert.DeserializeObject<T>(json, (JsonSerializerSettings?)null)` is the same as calling `JsonConvert.DeserializeObject<T>(json)`. See [here](https://github.com/JamesNK/Newtonsoft.Json/blob/7c3d7f8da7e35dde8fa74188b0decff70f8f10e3/Src/Newtonsoft.Json/JsonConvert.cs#L718-L721).

Reply:
I removed the extra function and added the default value to the new parameter.